OpenRA.Game/Map/CellLayer.cs

```diff
@@ -45,6 +45,19 @@ public void CopyValuesFrom(CellLayer<T> anotherLayer)
 			Array.Copy(anotherLayer.entries, entries, entries.Length);
 		}
 
+		public static CellLayer<T> CreatePrototype(Func<MPos, T> creator, Size size, TileShape tileShape)
+		{
+			var defaultCellInfoLayer = new CellLayer<T>(tileShape, size);
```

Comment:
This variable should probably be renamed since the current name is no longer applicable.